docs/book/user-guide/llmops-guide/evaluation/retrieval.md

```diff
@@ -0,0 +1,250 @@
+---
+description: See how the retrieval component responds to changes in the pipeline.
+---
+
+# Retrieval evaluation
+
+The retrieval component of our RAG pipeline is responsible for finding relevant
+documents or document chunks to feed into the generation component. In this
+section we'll explore how to evaluate the performance of the retrieval component
+of your RAG pipeline. We're checking how accurate the semantic search is, or in
+other words how relevant the retrieved documents are to the query.
+
+Our retrieval component takes the incoming query and converts it into a
+vector or embedded representation that can be used to search for relevant
+documents. We then use this representation to search through a corpus of
+documents and retrieve the most relevant ones.
+
+## Manual evaluation using handcrafted queries
+
```

Comment:
Potential diagram (Could use some work as to the arrows and precise component naming)

<img width="747" alt="image" src="https://github.com/zenml-io/zenml/assets/4792082/269a819f-450b-4815-bc77-d56a32b04614">